hibernate-core/src/main/java/org/hibernate/dialect/CockroachDialectQueryHints.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+
+package org.hibernate.dialect;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+class CockroachDialectQueryHints {
+
+	final private Pattern TABLE_QUERY_PATTERN = Pattern.compile(
```

Comment:
why not `static` too?

Reply:
Yeah, static would be better. Please create a PR :)